src/analysis/retail/priest/shadow/modules/tier/Tier31ShadowPriest4SetNormalizer.tsx

```diff
@@ -0,0 +1,77 @@
+import EventLinkNormalizer, { EventLink } from 'parser/core/EventLinkNormalizer';
+import {
+  CastEvent,
+  EventType,
+  GetRelatedEvents,
+  HasRelatedEvent,
+  DamageEvent,
+} from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+import TALENTS from 'common/TALENTS/priest';
+import SPELLS from 'common/SPELLS';
+import { TIERS } from 'game/TIERS';
+const HITS_TARGET = 'HitsTarget';
+const CONSUMES_T31_BUFF = 'ConsumesT31';
+
+const BUFFER_MS = 1000;
```

Comment:
On my example I used a buffer of 100. This is a much longer buffer than we'd normally use for insta-applied things, and this is longer than the minimum GCD so false positives are possible. Is there a reason you made it this large?

Reply:
It is a mistake, it should be shorter